index.html

```diff
@@ -40,6 +40,8 @@ <h3>Home</h3>
               <a class="dropdown-item" href="#" id="summerdayTheme">Summer Day Theme</a>
               <a class="dropdown-item" href="#" id="pumpkinSpiceTheme">Pumpkin Fall Theme</a>
               <a class="dropdown-item" href="#" id="abyssTheme">Abyss Theme</a>
+              <a class="dropdown-item" href="#" id="winterTheme">winter Theme</a>
```

Comment:
Capitalize Winter in the dropdown menu to keep capitalization consistent with other items in the menu ... `id="winterTheme">Winter Theme</a>`